core/cloudflow-akka/src/main/resources/akka-cluster-local.conf

```diff
@@ -1,6 +1,9 @@
 akka {
   actor {
     provider = "cluster"
+    serialization-bindings {
+      "cloudflow.akkastream.persistence.JacksonCborSerializable" = jackson-cbor
```

Comment:
I wonder if this should be a default? Is that something users will want to tune? Can you explain a bit more?

Reply:
See above. Jackson-cbor is recommended one. As I write in documentation, we are trying to put recommended defaults, if you want to overwrite, create your own application config, overwriting defaults. It works, I tested this